### PR TITLE
Added Instructions for Manual Testing appendix in DG

### PR DESCRIPTION
**Pull request by ulricolo7**

### CODE REVIEW

**Review by marraknas — COMMENTED** [review]
LGTM!

**Review by psh12320 — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -221,7 +221,7 @@ Edits the specified meeting with an Udder from the meetings list.
 
 **Format:** `editm INDEX [n/NAME] [st/DD-MM-YYYY] [et/DD-MM-YYYY] [l/LOCATION]`
 
-* Use the `list` command before editing any meeting!
+* Use the `meetings` command before editing any meetings!
```
Comment (XinghuaJulia) [inline docs/UserGuide.md:224]: Good bug catch!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -509,17 +509,147 @@ testers are expected to do more *exploratory* testing.
...
+
+    3. Test case: `find John`<br>
+       Expected: No filtering is done. Error details shown in the Result Display Panel.
+
+    4. Other incorrect delete commands to try: `find`, `find p/`, `...`<br>
+       Expected: Similar to previous.
+
+### Listing all Udders
+
+1. Listing all Udders from a filtered list of Udders
```
Comment (XinghuaJulia) [inline docs/DeveloperGuide.md:587]: Might this be misleading, do you mean `Listing all Udders after performing `find` command`

**Review by XinghuaJulia — COMMENTED** [review]
LGTM :DDDDD Overall, amazing work. Just a few minor nits.

**Pull request merged**